application/common/components/adapters/WorkdayIdStore.php

```diff
@@ -34,13 +34,14 @@ public function init()
                 ), 1532982562);
             }
         }
-        
+
         parent::init();
     }
-    
-    public static function getIdBrokerFieldNames()
+
+    public static function getFieldNameMap(): array
     {
         return [
+            // 'active' field isn't needed, since all Workday records returned are active.
```

Comment:
Was the phrase `all ID Store records returned are active` intentionally changed to `all Workday records returned are active`? If so, I'd be glad to learn why. It seems to imply that other ID Stores might also return inactive users, but I _think_ the design is that every ID Store will only return active users. I could be misremembering that detail, though.

Reply:
The Google adapter returns all users. I _think_ that's the only one.

Reply:
Ahh, good point. Okay, thanks.